src/core/scenario.jl

```diff
@@ -154,7 +156,8 @@ function Scenario(d::Dict; flex_hvac_from_json=false)
                                             emissions_factor_series_lb_CO2_per_kwh = 0,
                                             emissions_factor_series_lb_NOx_per_kwh = 0,
                                             emissions_factor_series_lb_SO2_per_kwh = 0,
-                                            emissions_factor_series_lb_PM25_per_kwh = 0
+                                            emissions_factor_series_lb_PM25_per_kwh = 0,
+                                            clean_energy_fraction_series = Float64[]
```

Comment:
oh! could just set this `clean_energy_fraction_series` to 0 for the off-grid case, and then would not have to check for off-grid within the electric_utility.jl code